developer-docs-site/docs/releases/testnet-release.md

```diff
@@ -17,5 +17,6 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
 
 |Release | Docker Image Tag | Docker SHA-256 | Branch | Commit Hash| Type |
 |---|---|---|---|---|---|
+|[Aptos Node Release v1.6.1](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.6.1) | [`aptos-node-v1.6.1`](https://hub.docker.com/layers/aptoslabs/validator/aptos-node-v1.6.1/images/sha256-3dd12caa0b7e67e2f3f7f3b29feb53405455526293df02f77b023b03d3ec26ee?context=explore) | `sha256:3dd12caa0b7e67e2f3f7f3b29feb53405455526293df02f77b023b03d3ec26ee` | [aptos-node-v1.6.1](https://github.com/aptos-labs/aptos-core/tree/aptos-node-v1.6.1)| `2ebf4ebeba428bf9c974d9b04c2358f6b8dc6eea` | Full Node and Validator  |
 |[Aptos Node Release v1.5.0](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.5.0) | [`aptos-node-v1.5.0`](https://hub.docker.com/layers/aptoslabs/validator/aptos-node-v1.5.0/images/sha256-3d9af9aadd4c1f2f866b3a6f79a3acaa2de372ef1fc3827757a6f4dc1803e557?context=explore) | `sha256:3d9af9aadd4c1f2f866b3a6f79a3acaa2de372ef1fc3827757a6f4dc1803e557` | [aptos-node-v1.5.0](https://github.com/aptos-labs/aptos-core/tree/aptos-node-v1.5.0)| `4fbd3d72130c361f54ded13c734400e5ed170bc1` | Full Node and Validator  |
-|[Aptos Node Release v1.4.5](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.4.5)| [`aptos-node-v1.4.5`](https://hub.docker.com/layers/aptoslabs/validator/aptos-node-v1.4.5/images/sha256-58c10b11ddd03c4fc998dda9ae08fdbbbca7dbe7ffea3d476bc38c2b6e54773c?context=explore) | `sha256:58c10b11ddd03c4fc998dda9ae08fdbbbca7dbe7ffea3d476bc38c2b6e54773c` | [aptos-node-v1.4.5](https://github.com/aptos-labs/aptos-core/tree/aptos-node-v1.4.5)| `940cc56e77baee84c1d81609591693733fabe125` | Validator  |
\ No newline at end of file
+|[Aptos Node Release v1.4.5](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.4.5)| [`aptos-node-v1.4.5`](https://hub.docker.com/layers/aptoslabs/validator/aptos-node-v1.4.5/images/sha256-58c10b11ddd03c4fc998dda9ae08fdbbbca7dbe7ffea3d476bc38c2b6e54773c?context=explore) | `sha256:58c10b11ddd03c4fc998dda9ae08fdbbbca7dbe7ffea3d476bc38c2b6e54773c` | [aptos-node-v1.4.5](https://github.com/aptos-labs/aptos-core/tree/aptos-node-v1.4.5)| `940cc56e77baee84c1d81609591693733fabe125` | Validator  |
```

Comment:
Looks like the diff here is a newline that github UI added. Funny how github UI doesn't show the newline